src/main/java/io/cdap/plugin/table/streaming/TableStreamingSource.java

```diff
@@ -55,45 +50,48 @@
   "intervals. The primary use case for this plugin is to send it to a Joiner plugin to provide lookup-like " +
   "functionality.")
 public class TableStreamingSource extends StreamingSource<StructuredRecord> {
-  private final Conf conf;
+  private final TableStreamingSourceConfig conf;
```

Comment:
Can we rename to `config` so it's consistent with what's in other plugins.

Reply:
Done.